Bugsnag/Assets/Bugsnag/Runtime/Payload/Event.cs

```diff
@@ -162,6 +162,8 @@ internal void AddAndroidProjectPackagesToEvent(string[] packages)
             _androidProjectPackages = packages;
         }
 
+        private bool _originalUnhandledState;
```

Comment:
We already have a ticket (PLAT-9619) that we're not setting `unhandledOverridden` in the severity reason. I think we should consider setting this as part of this change.